js/stringProcessing/findKeywordInUrl.js

```diff
@@ -0,0 +1,22 @@
+/** @module stringProcessing/findKeywordInUrl */
+
+var keywordRegex = require( "../stringProcessing/stringToRegex.js" );
+/**
+ *
+ * @param {string} url The url to check for keyword
+ * @param {string} keyword The keyword to check if it is in the URL
+ * @returns {boolean} If a keyword is found, returns true
+ */
+module.exports = function( url, keyword ) {
+	var keywordFound = false;
+	if ( typeof keyword !== "undefined" ) {
```

Comment:
We decided to keep these checks out of the analyses and string processing so this one should be removed as well.